product_profile/models/product_profile.py

```diff
@@ -9,9 +9,7 @@
 from odoo import _, api, fields, models
 from odoo.exceptions import UserError
 
-PROFILE_MENU = _(
-    "Sales > Configuration \n> Product Categories and Attributes" "\n> Product Profiles"
-)
+PROFILE_MENU = _("Sales > Configuration \n> Products" "\n> Product Profiles")
```

Comment:
You can't translate without a context. Thus, this won't work. Please just hardcode this wherever needed.

Check this out: https://www.odoo.com/documentation/16.0/developer/howtos/translations.html#explicit-exports

Reply:
Noted / changed